BUILDING_ON_WINDOWS.md

```diff
@@ -82,19 +93,33 @@ Note: This installation will take about 200 MB of disk space.
 
 Install [conan 2](https://conan.io/downloads.html) and make sure it's in your `PATH` (default setting).
 
+<details>
+   <summary>How to add conan 2 to PATH</summary>
```

Comment:
This adds python's scripts to `PATH`. Maybe this should read something like "Installing with pip" and include the `pip install`. Furthermore, I thought default python installations would add themselves to `PATH` by default.

Reply:
Yeah, maybe its pip specific. I don't know if default python installations add themselves to PATH by default, I think I manually added conan 2 to PATH (don't remember if it was there before). 

I'm probably just going to commit Felanbird's suggestion to remove it after I ask about something. 

The reason I haven't committed it yet because I'm not sure if I should resolve a conversation (by doing a commit) when I still have a question about a related topic.